pkg/networkservice/mechanisms/vxlan/common.go

```diff
@@ -60,6 +61,13 @@ func addDel(ctx context.Context, conn *networkservice.Connection, vppConn api.Co
 			NextName: "ethernet-input",
 		}
 
+		if mechanism.SrcIP().To4() == nil {
+			addNextNode = &vpe.AddNodeNext{
+				NodeName: "vxlan6-input",
+				NextName: "ethernet-input",
+			}
+		}
+
```

Comment:
Nice catch!

Whenever you are comfortable switching this from draft, I'm fine merging it :)